CONTRIBUTING.md

```diff
@@ -119,3 +125,22 @@ Tool Recommendation: Firefox Browser (easier to make screenshots)
 ```
 !["Alt test for accessibility"](/static/img/name-of-the-image.png)
 ```
+
+### Search
+
+We use Algolia to build our index. In order to build an index for search:
+
+1. sign up for an account on https://www.algolia.com/users/sign_in
```

Comment:
@CameronGray1210 and I tried to follow these steps, but there's a bunch of missing info here.  Going through the Algolia site to create our index is more complicated than what is listed here.  I'd like to see this expanded.  For example, unless we upload a sample data set we can't move forward with the process.

Reply:
ya but if i copy the whole site info on here will be long as well, thats why i provided links
Since you only create the account once.

Reply:
i was under the impression if your site is live, it crawls the website automatically building the index,  I believe.  That is what the script does.

Reply:
I'm not sure if you need to upload the whole website onto Algo.. thats only if you have them build it for you

Reply:
we are building our own index